Proteomics/Modifications/ModificationGeneral.cs

```diff
@@ -14,9 +14,22 @@ public class ModificationGeneral
         public string ModificationType { get; private set; }
         public string FeatureType { get; private set; }
         public ModificationMotif Target { get; private set; }
-        public ModLocationOnPeptideOrProtein? Position { get; private set; }
+        public string Position { get; private set; }
         public ChemicalFormula ChemicalFormula { get; private set; }
-        public double? MonoisotopicMass { get; private set; }
+        private double? monoisotopicMass = null;
```

Comment:
Could you use `_MonoisotopicMass` as the tracking field name instead?

Reply:
What is a tracking field?

Reply:
Private field used in the getter or setter methods of a property.

Reply:
You can type CTRL+R+R to do a quick refactor of everywhere the variable name occurs, btw.

Reply:
Try explaining again and pretend I'm in kindergarten. Also, thanks for the tip. That will be handy.

Reply:
doesn't "this" do that? 
![image](https://user-images.githubusercontent.com/16841846/42841617-4135cdf4-89d0-11e8-9542-ed7086b42b27.png)